src/compiler.jl

```diff
@@ -6059,10 +6059,8 @@ function to_tape_type(Type::LLVM.API.LLVMTypeRef)::Tuple{DataType,Bool}
             return UInt32, false
         elseif N == 64
             return UInt64, false
-        elseif N == 128
-            return UInt128, false
         else
-            error("Can't construct tape type for integer of width $N")
```

Comment:
Why delete the branch for UInt128?

Reply:
Otherwise we assert in LLVM with getActiveBits() <= 64 && "Too many bits for uint64_t"